DESCRIPTION

```diff
@@ -27,12 +27,14 @@ BugReports: https://github.com/r-lib/pkgdown/issues
 Depends: 
     R (>= 3.1.0)
 Imports: 
+    bslib,
```

Comment:
too many dependencies according to CRAN now

Reply:
Can you make an issue to track this please?

Reply:
#1559 1559